### PR TITLE
feat(azure): document nRF Cloud Assisted GPS Location Service

### PR DESCRIPTION
**Pull request by coderbyheart**
See https://github.com/NordicSemiconductor/asset-tracker-cloud-azure-js/pull/154

### CODE REVIEW

File: docs/azure/nRFCloudLocationServices.rst [hunk 1]
```diff
@@ -42,3 +42,42 @@ Store the service key into the key vault as follows:
    az keyvault secret set --vault-name ${APP_NAME:-nrfassettracker} \\
      --name nrfCloudCellLocationServiceKey \\
      --file *location of your Cell Location Service Key file*
+
+Assisted GPS Location Service
+*****************************
+
+You can enable your devices to request assisted GPS data using `nRF Cloud's Assisted GPS Location Service API <https://api.nrfcloud.com/v1#tag/Assisted-GPS>`_ in your deployment.
```
Comment (peknis) [inline docs/azure/nRFCloudLocationServices.rst]:
```suggestion
You can enable your devices to request assisted GPS (A-GPS) data using `nRF Cloud's Assisted GPS Location Service API <https://api.nrfcloud.com/v1#tag/Assisted-GPS>`_ in your deployment.
```

File: docs/azure/nRFCloudLocationServices.rst [hunk 2]
```diff
@@ -42,3 +42,42 @@ Store the service key into the key vault as follows:
...
+            nrfCloudTeamId=*your team ID*
+
+This command enables the ``agpsQueuedDeviceRequestsHandler`` function to resolve A-GPS requests from devices using the nRF Cloud Assisted GPS Location Service API.
+
+Store the service key into the key vault as follows:
+
+.. parsed-literal::
+   :class: highlight
+
+   # Grant the current user set permission to the key vault secrets
```
Comment (peknis) [inline docs/azure/nRFCloudLocationServices.rst:76]: user set or users?

File: docs/azure/nRFCloudLocationServices.rst [hunk 3]
```diff
@@ -42,3 +42,42 @@ Store the service key into the key vault as follows:
...
+   USER_OBJECT_ID=`az ad signed-in-user show --query objectId -o tsv`
+   az keyvault set-policy --name ${keyVaultName} --object-id ${USER_OBJECT_ID} --secret-permissions set
+
+   # Store the API key
+   az keyvault secret set --vault-name ${APP_NAME:-nrfassettracker} \\
+     --name nrfCloudAGPSLocationServiceKey \\
+     --file *location of your Assisted GPS Location Service Key file*
```
Comment (peknis) [inline docs/azure/nRFCloudLocationServices.rst:83]: Maybe A-GPS to make this a bit shorter.

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by wiba-nordic — APPROVED** [review]

File: docs/azure/nRFCloudLocationServices.rst (hunk 2 above)
Comment (coderbyheart) [reply docs/azure/nRFCloudLocationServices.rst:76]: It's just one user.

File: docs/azure/nRFCloudLocationServices.rst (hunk 3 above)
Comment (coderbyheart) [reply docs/azure/nRFCloudLocationServices.rst:83]: But this is the name of the service: 
image

File: docs/azure/nRFCloudLocationServices.rst (hunk 2 above)
Comment (peknis) [reply docs/azure/nRFCloudLocationServices.rst:76]: This is still a bit unclear to me. Will the user be given a permission to use the key vault secrets or set some permissions?

File: docs/azure/nRFCloudLocationServices.rst (hunk 3 above)
Comment (peknis) [reply docs/azure/nRFCloudLocationServices.rst:83]: Fair enough.

File: docs/azure/nRFCloudLocationServices.rst (hunk 2 above)
Comment (coderbyheart) [reply docs/azure/nRFCloudLocationServices.rst:76]: Azure keyvaults have different permissions, this command grants the user the "set" permission, which is later used in the command:

```
az keyvault secret set ...
                   ^^^
```

Comment (peknis) [reply docs/azure/nRFCloudLocationServices.rst:76]: Okay. It just sounded weird. Set permission it is then.

**Review by peknis — APPROVED** [review]

**Pull request merged**